index.html

```diff
@@ -157,6 +157,19 @@ <h3 class="refs__links__title">Volby do Představenstva FIT++<br>na volební obd
         <a class="refs__links__link" href="/documents/volby2018/volby.pdf"><i class="refs__links__link__logo ion ion-ios-cloud-download"></i> Informace k volbám</a>
         <a class="refs__links__link" href="/documents/volby2018/kandidatky.pdf"><i class="refs__links__link__logo ion ion-ios-cloud-download"></i> Kandidátky</a>
         <a class="refs__links__link" href="/documents/volby2018/vysledky.pdf"><i class="refs__links__link__logo ion ion-ios-cloud-download"></i> Výsledky voleb</a>
+
+        <h3 class="refs__links__title">Hlasování Per Rollam ohledně výroční zprávy za rok 2021</h3>
+        <p class="refs__links__text">Představenstvo FIT++ schvaluje výroční zprávu FIT++ za rok 2021.</p>
+        <a class="refs__links__link" href="/documents/vyrocniZpravy/vz_fitpp_2021_navrh1.pdf"</a><i class="refs__links__link__logo ion ion-ios-cloud-download"></i>Výroční zpráva</a>
+        <details>
```

Comment:
V tuhle chvíli nejsou napsané žádné styly pro `details` tag. Zkus prosím přidat styly (a zarovnat tyhle prvky třeba do tabulky), nebo tam jen hoď tohle

```html
<p class="refs__links__text">
    Hashe md5 pro hlasování představenstva
    <br>
    (Byl použit program md5sum (GNU coreutils) 9.0)
    <br>
    <strong>Petr Šťastný:</strong> 2b8e560ec3ebf70fcd986c74a54f2aa8
    <br>
    <strong>Jakub Horák:</strong> ae47bb3d47a0a31f17d9fe23a28cd489
    <br>
    <strong>Damián Baraník:</strong> 03889d86abb80fc9db8917f2d1a3e068
    <br>    
    <strong>Jaroslav Kříž:</strong> 1fcb5777ecd89838c9ce588e630ae718
    <br>
    <strong>Lukáš Hypša:</strong> 42450a2911029e2b0a273f85614664f5
</p>
```

ať to drží původní styl a nejde to mimo zarovnání 🙏

Pak to bude vypadat takhle

![image](https://user-images.githubusercontent.com/15214494/154706358-ba547809-3143-484c-a867-0a5cc666e666.png)

Reply:
Yeetnul jsem tam tvoji classu

Reply:
Trochu kodérskej parkour, ale good enough :ok_hand: